api-list.md

```diff
@@ -1,5 +1,24 @@
 # API
 
+## Verbs
+
+- `GET` - Read
+- `POST` - Create
+- `DELETE` - remove/deactivate
+- `PUT` - update by **setting** data
+- `PATCH` - update by **adding** data
```

Comment:
Addition of PATCH verb so that we can sometimes implement adding data, as well as setting data

Reply:
Yeah, I wasn't sure about when/if to use PATCH, but I like this as a convention.